packages/@okta/vuepress-theme-prose/layouts/Layout.vue

```diff
@@ -195,6 +213,12 @@ export default {
     this.$on("toggle-tree-nav", event => {
       that.appContext.isTreeNavMobileOpen = event.treeNavOpen;
     });
+    
+    // Delay height adjustment on mount to ensure the header element is fully rendered.
+    setTimeout(() => {
```

Comment:
I guess we don't have a good way to avoid `setTimeout` approach...

Reply:
Yes, especially when calculating height in javascript.